server/src/test/java/io/crate/integrationtests/ShowIntegrationTest.java

```diff
@@ -252,31 +256,25 @@ public void testWeirdIdentifiers() throws Exception {
         ensureYellow();
 
         execute("SHOW CREATE TABLE with_quote");
-        assertRow("CREATE TABLE IF NOT EXISTS \"doc\".\"with_quote\" (\n" +
-                  "   \"\"\"\" TEXT\n" +
-                  ")\n" +
-                  "CLUSTERED INTO 1 SHARDS");
+        assertFirstRow().startsWith("CREATE TABLE IF NOT EXISTS \"doc\".\"with_quote\" (\n" +
+                                    "   \"\"\"\" TEXT\n" +
+                                    ")\n" +
+                                    "CLUSTERED INTO 1 SHARDS");
 
     }
 
-    private void assertRow(String expected) {
-        assertEquals(1L, response.rowCount());
-        try {
-            assertThat(((String) response.rows()[0][0]), startsWith(expected));
-        } catch (Throwable e) {
-            String msg = String.format(Locale.ENGLISH, "Row does not start with expected string:%n%n" +
-                                                       "Expected: %s%nActual: %s%n", expected, response.rows()[0][0]);
-            throw new AssertionError(msg);
-        }
+    private AbstractStringAssert<?> assertFirstRow() {
```

Comment:
Not pretty, but works in this case.

Reply:
I don't think it's ugly, It looks pretty nice this way imho.
Just, please move the private method to the end of the class, after the test methods.

Reply:
👍